GNUmakefile

```diff
@@ -42,6 +42,9 @@ vet:
 		exit 1; \
 	fi
 
+lint: get-test-deps
+	golint ./...
+
```

Comment:
On a similar note, I think it'd be pretty useful if this eventually got called from `make testall` 
Otherwise you have to know what individual commands to run in order to reproduce what the CI will do

Reply:
Done I think.